tap_linkedin_ads/client.py

```diff
@@ -141,12 +149,46 @@ def __init__(self,
                           max_tries=5,
                           factor=2)
     def __enter__(self):
-        self.__verified = self.check_access_token()
+        self.__verified = self.get_access_token()
         return self
 
     def __exit__(self, exception_type, exception_value, traceback):
         self.__session.close()
 
+    @backoff.on_exception(backoff.expo,
+                          Server5xxError,
+                          max_tries=5,
+                          factor=2)
+    def get_access_token(self):
```

Comment:
The name `get_access_token` as a method on a class insinuates that it would return an access token. What would you think about naming this something like `fetch_and_set_access_token` or something of the like? A little more wordy, but it describes the operation a little more clearly.

Reply:
correct i have renamed the method name